Update set-up-multi-factor-authentication.md (#609)
MicrosoftDocs/OfficeDocs-O365ITPro: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md

All Office 2016 client applications support MFA through the use of the Active Directory Authentication Library (ADAL). This means that app passwords aren't required for Office 2016 clients. However, you need to make sure your Office 365 subscription is enabled for ADAL, or modern authentication.

1. To enable modern authentication, go to the admin center at <a href="https://go.microsoft.com/fwlink/p/?linkid=2024339" target="_blank">https://admin.microsoft.com</a>. 

> 2. Turn On the Preview Mode by Selecting Try the preview toggle located at the top of the Home page.

Reviewer: ```suggestion
2. Turn on preview mode by selecting **Try the preview** toggle located at the top of the Home page.
```

> 2. Turn On the Preview Mode by Selecting Try the preview toggle located at the top of the Home page.

Reviewer 2: ```suggestion
2. Turn on the Preview Mode by selecting **Try the preview** toggle located at the top of the Home page.
```

Reviewer 3: UI has changed. Step 2 should now read:
Turn on the new admin center by selecting the "Try the new admin center" toggle located at the top of the Home page.

Reviewer 3: Turn on the new admin center by selecting **Try the new admin center** toggle located at the top of the Home page.

> 2. Choose **Settings** \> **Services & add-ins** and then choose **Modern authentication** from the list.

Reviewer: ```suggestion
2. Choose **Settings** > **Services & add-ins** and then choose **Modern authentication** from the list.
```

> 2. Choose **Settings** \> **Services & add-ins** and then choose **Modern authentication** from the list.

Reviewer 2: ```suggestion
3. Choose **Settings** \> **Services & add-ins** and then choose **Modern authentication** from the list.
```

> 3. Check the **Enable modern authentication** box in the **Modern authentication** panel. 

Reviewer 2: ```suggestion
4. Check the **Enable modern authentication** box in the **Modern authentication** panel. 
```

All Office 2016 client applications support MFA through the use of the Active Directory Authentication Library (ADAL). This means that app passwords aren't required for Office 2016 clients. However, you need to make sure your Office 365 subscription is enabled for ADAL, or modern authentication.

1. To enable modern authentication, go to the admin center at <a href="https://go.microsoft.com/fwlink/p/?linkid=2024339" target="_blank">https://admin.microsoft.com</a>. 

> 2. Turn on the Preview Mode by selecting **Try the preview** toggle located at the top of the Home page.

Reviewer 4: ```suggestion
2. Turn on the new admin center by selecting **Try the new admin center** toggle located at the top of the Home page.
```
